src/components/details.css

```diff
@@ -55,10 +48,67 @@
 .details-container {
   background-color: rgb(0, 59, 86);
   height: 100vh;
-
 }
 
 .not-found {
+  font-size: 40px;
   text-align: center;
-  color: #355087;
-}
\ No newline at end of file
+  color: red;
+  margin-top: 5%;
+} */
+
```

Comment:
- [OPTIONAL] Kindly remove all commented-out codes from your CSS file. Removing unused codes makes your code look clean and readable. 💻 🖱️